MAUI/Calendar/date-navigation.md

```diff
@@ -70,6 +70,9 @@ this.Calendar.AllowViewNavigation = true;
 {% endhighlight %}
 {% endtabs %}
 
+## Navigation to adjacent months by using leading/trailing dates
+The next and previous months can be navigated by tapping the leading and trailing dates of Calendar control.Tapping on leading dates will navigate to previous month and tapping on trailing dates will navigate to the next month. In `SfCalendar`, this feature can be enabled or disabled by changing this property [NavigateToAdjacentMonth]().
```

Comment:
**Navigation to Adjacent Months Using Leading/Trailing Dates**

You can navigate to the next or previous month by tapping on the leading or trailing dates within the Calendar control. When you tap on a leading date, the calendar will navigate to the previous month, tapping on a trailing date will navigate to the next month. In SfCalendar, this functionality can be enabled or disabled by modifying the [NavigateToAdjacentMonth](https://github.com/syncfusion-content/maui-docs/pull/2554/files#) property.

Reply:
Changed the description as mentioned